dace/codegen/compiler.py

```diff
@@ -297,6 +298,29 @@ def unique_flags(flags):
     return set(re.findall(pattern, flags))
 
 
+def file_changed(filename: str, file_contents: str):
+    # If file did not exist before, it has changed
```

Comment:
Small nitpick: maybe it's more accurate to call this function something like `identical_file_exists`, to make it clear that it covers both whether the file exists at all, and that the content should be the same? This of course requires negating the return value.

Reply:
Done